2-mandatory/9-cheap-diner.js

```diff
@@ -19,7 +19,17 @@ chosenMeal(emptyArray)
 Should give the answer "Nothing :("
 **/
 
-function chooseMeal(mealArray) {}
+function chooseMeal(mealArray) {
+  if(mealArray.length === 0 ){
+    return "Nothing"
+  }
+  if(mealArray.length === 1) {
+    return mealArray[0].name;
+  }
+  let newArray = [...mealArray]
+  newArray.sort((a ,b) => a.price - b.price)
+  return newArray[1].name
+}
```

Comment:
I like your solution to this problem. Nice, neat and understandable. This is one of the exercises I was not able to solve however reading your solution has given me ideas on how to approach this problem. Really awesome work Dona.